contract/package.json

```diff
@@ -14,8 +14,7 @@
     "build": "yarn build:deployer",
     "build:deployer": "rollup -c rollup.config.mjs",
     "test": "ava",
-    "lint": "eslint '**/*.js'",
-    "lint:types": "tsc -p jsconfig.json",
```

Comment:
we don't do `lint:types` any more?

Reply:
nothing was running `lint:types`. agoric-sdk had it that way for backwards compatibility. That scheme requires something like `run-s` to lint all the `lint:foo` targets.

When you want to lint types, I think `yarn tsc` is handy enough. Fewer chars. And it's fast enough that when you want to eslint, it doesn't hurt to wait for tsc first